code-studio/src/css/plc/preview_evaluation.scss

```diff
@@ -0,0 +1,12 @@
+.preview_evaluation {
+  display: flex;
```

Comment:
Flex is supported enough on IE 11. On IE 9/10 it degrades to

![image](https://cloud.githubusercontent.com/assets/14246419/15029481/a066483c-1202-11e6-95b8-dd57b7e44e61.png)

So still functional